rdtools/test/filtering_test.py

```diff
@@ -521,4 +519,4 @@ def test_hour_angle_filter():
 
     # Check that the result is the correct boolean Series
     expected_result = pd.Series([False, False, True, True, True], index=index)
-    pd.testing.assert_series_equal(result, expected_result)
+    pd.testing.assert_series_equal(result, expected_result, check_names=False)
```

Comment:
What caused the need for this change?

Reply:
In the newest pvlib version 0.11.1 there was a bug fix for the hour_angle() function (https://pvlib-python.readthedocs.io/en/stable/whatsnew.html). 

The series that's returned does have a new argument `Name: equation_of_time`. 
![image](https://github.com/user-attachments/assets/ddf4883f-45bd-4c82-a9fe-cee58adfbee4)

In order to make the test run for older pvlib versions in which the name attribute is not present, I decided to not add the name to the expected result series but rather don't check for series names.